Engine/source/shaderGen/shaderGen.cpp

```diff
@@ -49,6 +49,7 @@ MODULE_BEGIN( ShaderGen )
 
 MODULE_END;
 
+String ShaderGen::smCommonShaderPath(Con::getVariable("$Core::CommonShaderPath", "shaders/common"));
```

Comment:
It actually does set it when shaderGen itself is initialized, here: https://github.com/GarageGames/Torque3D/pull/2055/files#diff-2d6f2326413f62282d66e1c77566a3c7R129

So it works fine, but you mentioning it does highlight how silly having the getVariable in that initial definition of the var is. It should just be 
`String ShaderGen::smCommonShaderPath("shaders/common");`

Got too eager with the copy/paste, there ;)

I'll get that updated.